### PR TITLE
[BACKLOG-36953] Page Zoom (AA) - Browse Files - Schedule and Run in Background - Select Folder

### PR DESCRIPTION
**Pull request by davidmsantos90**
@pentaho/wcag @pentaho/millenniumfalcon please review

**Note:** wasn't able to get the tree items to show an ellipsis on overflow. (but they have tooltips)

### CODE REVIEW

**Review by mbrasil — APPROVED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 45s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **398**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-882/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 39s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **398**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-882/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: widgets/src/main/java/org/pentaho/mantle/client/dialogs/folderchooser/FolderTree.java [hunk 1]
```diff
@@ -426,16 +426,20 @@ public void onFetchRepositoryFileTree( RepositoryFileTree fileTree, List<Reposit
   private void fixLeafNodes() {
     List<FolderTreeItem> allNodes = getAllNodes();
     for ( FolderTreeItem treeItem : allNodes ) {
+      LeafItemWidget leafWidget;
```
Comment (Pancho7) [inline widgets/src/main/java/org/pentaho/mantle/client/dialogs/folderchooser/FolderTree.java:429]: update copyright date only

**Review by Pancho7 — APPROVED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 43s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **398**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-882/4/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**